EasyPlantsApp/app/src/main/java/com/example/gamingpc/easyplants/MainActivity.java

```diff
@@ -57,6 +68,73 @@ protected void onCreate(Bundle savedInstanceState) {
         setContentView(R.layout.activity_main);
 
         setup();
+
+        // Write a message to the database
+        FirebaseDatabase database = FirebaseDatabase.getInstance();
+        DatabaseReference myRef = database.getReference("message");
+
+        myRef.setValue("Hello, World!");
+        //notificationCall();
+
+        //Retrieve humidity value from firebase database
+        //TODO change path of child to read from message_list
+        DatabaseReference humidityValue = FirebaseDatabase.getInstance().getReference().child("HumidityTest").child("humidity_value");
+
+
+        //keep track of humidity value in Firebase
+        humidityValue.addValueEventListener(new ValueEventListener() {
+            @Override
+            public void onDataChange(DataSnapshot dataSnapshot) {
+                Integer humidity = dataSnapshot.getValue(Integer.class);
+                Log.d(TAG, Integer.toString(humidity));
+
+                //notify user if humidity is under or over a certain threshold
+                //TODO retrieve threshold from firebase
+                if(humidity < 40){
```

Comment:
Hi, thanks for this! This looks great!

Please remember to refactor this section of the code for sprint 3. We'll also need to differentiate between an old humidity value and a newer humidity value (e.g. not alert the user if the humidity threshold is below value a few months ago)


```suggestion
                if(humidity < 40){
                notificationCall("Water your plant!");
                }
                else if (humidity > 60){
                    notificationCall("Your plant has too much water.");
                }
            }
```